chart/values.yaml

```
@@ -1153,6 +1153,8 @@ redis:
   affinity: {}
   tolerations: []
 
+  uid: null
```

```suggestion
  uid: ~
```

might work as null ? @jedcunningham @dstandish WDYT ?

---

Sure, http://yaml.org/spec/1.2-old/spec.html#id2805071